with-react-three-fiber/package.json

```diff
@@ -1,14 +1,12 @@
 {
   "dependencies": {
     "expo": "^43.0.0",
-    "expo-gl": "~11.0.3",
-    "expo-three": "~5.7.0",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
+    "react": "18.0.0-rc.0",
```

Comment:
React Native doesn't support the unreleased React version yet.

Reply:
I see, probably closing this then. Is there a timeline for this or is it too early to know?

Reply:
Well after React 18 comes out, React Native will add support (they have a pretty unpredictable release schedule), then Expo will bump to React Native, so it could be a while. We could potentially merge before Expo has full support for React 18 but it's hard to say.